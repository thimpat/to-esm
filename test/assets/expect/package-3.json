{
  "name": "something",
  "main": "./example/cjs/demo.cjs",
  "module": "./generated/esm/demo.mjs",
  "type": "module",
  "exports": {
    ".": {
      "require": "./given/demo-test-14.cjs",
      "import": "./actual/demo-test-14.mjs"
    }
  }
}