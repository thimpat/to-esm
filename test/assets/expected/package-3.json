{
  "name": "something",
  "main": "./example/cjs/demo.cjs",
  "module": "./actual/demo-test-14.mjs",
  "type": "module",
  "exports": {
    ".": {
      "require": "./given/demo-test-14.cjs",
      "import": "./actual/demo-test-14.mjs"
    }
  }
}